### PR TITLE
refactor(provider/kubernetes) Update the dependency jars and fix config problem.

### PR DESCRIPTION
**Pull request by chlung**
We prefer small, well tested pull requests.

Please refer to [Contributing to Spinnaker](https://spinnaker.io/community/contributing/).

When filling out a pull request, please consider the following:

* Follow the commit message conventions [found here](http://www.spinnaker.io/v1.0/docs/how-to-submit-a-patch).
* Provide a descriptive summary for your changes.
* If it fixes a bug or resolves a feature request, be sure to link to that issue.
* Add inline code comments to changes that might not be obvious.
* Squash your commits as you keep adding changes.
* Add a comment to @spinnaker/reviewers for review if your issue has been outstanding for more than 3 days.

Note that we are unlikely to accept pull requests that add features without prior discussion. The best way to propose a feature is to open an issue first and discuss your ideas there before implementing them.

In this PR,
The gradle build will reference kuberentes client java jars in maven repo.  I also fix a config bug.

### CODE REVIEW

**Comment by chlung** [comment]
@lwander Please you review.  Thanks.

File: clouddriver-kubernetes/clouddriver-kubernetes.gradle [hunk 1]
```diff
@@ -6,6 +6,6 @@ dependencies {
   compile spinnaker.dependency('bootActuator')
   compile spinnaker.dependency('bootWeb')
 
-  //FIXME: Remove the following line when k8s client java api jar is available in maven repo
-  compile fileTree(dir: 'kubernetes-client-java-libs', include: '*.jar')
+  compile 'io.kubernetes:client-java-api:0.1'
```
Comment (lwander) [inline clouddriver-kubernetes/clouddriver-kubernetes.gradle]: You only need `-util`, and later (next release) you'll only need `client-java`

File: clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesApiClientConfig.groovy [hunk 2]
```diff
@@ -34,10 +34,6 @@ public class KubernetesApiClientConfig extends Config {
   }
 
   public ApiClient getApiCient() throws Exception {
-    if (kubeconfigFile) {
-      return Config.defaultClient()
-    } else {
-      return fromConfig(kubeconfigFile)
-    }
+    return (kubeconfigFile ? fromConfig(kubeconfigFile) : Config.defaultClient())
```
Comment (lwander) [inline clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesApiClientConfig.groovy]: :+1:

**Comment by chlung** [comment]
@lwander Update per you feedback.  I also removed these previously  included jar files.  Please review.  thanks.

**Pull request merged**